tools/releasing/create_source_release.sh

```diff
@@ -77,10 +77,29 @@ rsync -a \
   --exclude ".travis.yml" \
   . flink-kubernetes-operator-${RELEASE_VERSION}
 
+# Package sources
 tar czf ${RELEASE_DIR}/flink-kubernetes-operator-${RELEASE_VERSION}-src.tgz flink-kubernetes-operator-${RELEASE_VERSION}
 gpg --armor --detach-sig ${RELEASE_DIR}/flink-kubernetes-operator-${RELEASE_VERSION}-src.tgz
+
+# Package helm chart
+commit_hash=$(git log -1 --pretty=format:%h)
+
+# TODO: We might want to be more specific here later on what to replace
+perl -pi -e "s#^  repository: .*#  repository: ghcr.io/apache/flink-operator#" helm/flink-operator/values.yaml
+perl -pi -e "s#^  tag: .*#  tag: ${commit_hash}#" helm/flink-operator/values.yaml
```

Comment:
We are using commit hash as the image tag, not the `${RELEASE_VERSION}`. Right?

Reply:
exactly, we publish the image build with the short commit hash

Reply:
We can certainly improve this once we get some more experience with this, but currently this is the approach we can use for this preview release candidate.

Reply:
It seems that the image tag in [rc2](https://dist.apache.org/repos/dist/dev/flink/flink-kubernetes-operator-0.1.0-rc2/) is `260df17`. But I could not find this commit in release-0.1. Is this the expected behavior?

Reply:
The commit recorded in the RC is the commit of the release tag: https://github.com/apache/flink-kubernetes-operator/tree/release-0.1.0-rc2

Reply:
You are correct.